web/src/containers/activity/All.js

```diff
@@ -55,8 +34,7 @@ const All = ({ add, first, other, remove }) => {
 All.propTypes = {
   add: PropTypes.func.isRequired,
   first: PropTypes.object.isRequired,
-  other: PropTypes.arrayOf(PropTypes.object).isRequired,
-  remove: PropTypes.func.isRequired
+  other: PropTypes.arrayOf(PropTypes.object).isRequired
 };
 
 const mapStateToProps = state => {
```

Comment:
We don't need the distinction between the first and other activities anymore, so it should be fine to replace `mapStateToProps` with something like:

```js
mapStateToProps = state => ({ activities: selectAllActivities(state) });
```

And update the rest of this component accordingly (e.g., replace `[first, ...other]` with just `activities`).